dev-db/influxdb/files/influxdb.conf.d

```diff
@@ -0,0 +1,3 @@
+# /etc/conf.d/influxdb: configuration for /etc/init.d/influxdb
+
+INFLUXD_OPTS=""
```

Comment:
Does this package actually include any useful options or are you just copying some boilerplate?